sdk/monitor/Azure.Monitor.Query/CHANGELOG.md

```diff
@@ -1,14 +1,6 @@
 # Release History
 
-## 1.2.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+## 1.2.0 (2023-05-09)
```

Comment:
Since some text is required in the entry, we could say the following. I couldn't think of any changes to the public API surface.

```suggestion


## 1.2.0 (2023-05-09)

Features Generally Available from v1.2.0-beta.1

```